### PR TITLE
Added DeleteCompleteFilesTask

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-716

### Change description ###
Added DeleteCompleteFilesTask - just empty implementation, logic will be added in next PRs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5280006&bid=17287492)

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5280006&bid=17287492)

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import static uk.gov.hmcts.reform.bulkscanprocessor.util.TimeZones.EUROPE_LONDON;
+
+@Service
+@ConditionalOnProperty(value = "scheduling.task.delete-complete-files.enabled")
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java:13]: missing main application config here?

**Comment by apereverzin** [comment]
This PR is for configuring the scheduled task, not for the task itself

**Comment by nitinprabhu** [comment]
> This PR is for configuring the scheduled task, not for the task itself

so why not create the task first ?

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/SchedulerConfigTest.java [hunk 2]
```diff
@@ -24,7 +24,9 @@
     properties = {
         "scheduling.task.scan.enabled=true",
         "scheduling.task.reupload.enabled=true",
-        "scheduling.task.notifications_to_orchestrator.enabled=true"
+        "scheduling.task.notifications_to_orchestrator.enabled=true",
+        "scheduling.task.delete-complete-files.enabled=true",
+        "scheduling.task.delete-complete-files.cron: */1 * * * * *"
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/SchedulerConfigTest.java:29]: `=`

Comment (rkondratowicz) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/SchedulerConfigTest.java:29]: isn't `*/1` same as `*`?

**Comment by apereverzin** [comment]
Will be implemented in another PR

**Pull request closed without merging**